components/block/BlockDetails.tsx

```diff
@@ -251,6 +249,33 @@ const BlockDetails = () => {
             </Typography>
             <Divider variant="middle" className={classes.divider} />
           </Grid>
+
+          <Grid item xs={12} >
+            <Typography variant="body2" component="h2" gutterBottom>
+              Signers
+            </Typography>
+            {data.block && data.block.signers
+              ?
+              data.block.signers.map((row: any, index: number) => {
+                return (
+                  <span key={index}>
+
+                    <Typography variant="body2" gutterBottom >
```

Comment:
We better show this as a list indicating if the validator has signed the block. The current implementation doesn't show whether a validator is missing in this block.

Two approaches:
1. List as a table with the first column the validator, second column whether it's up or down
2. Two tabs list out all up validators and down validators.

Reply:
Ok! I have updated the code and added Tab List with Signed & Missed Blocks